cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CompileTask.java

```diff
@@ -61,6 +62,8 @@ public void execute(Project project) {
         // Print the source
         this.out.println("\t" + sourceName);
 
+        System.setProperty(CentralClientConstants.ENABLE_OUTPUT_STREAM, "false");
```

Comment:
The progress bar should show in the stdout in the case of commands.

Reply:
Changed to true as discussed